util/chunk/chunk.go

```diff
@@ -277,6 +278,60 @@ func (c *Chunk) AppendPartialRow(colIdx int, row Row) {
 	}
 }
 
+// PreAlloc4Row pre-allocates the memory space for a Row.
+// The null elem info will be pre-written.
```

Comment:
1. s/elem info/bitmap/
2. should the schema of `row` and `c` be the same?

Reply:
Yes, but it's should be promised by the caller.